packages/components/src/SubHeaderBar/SubHeaderBar.component.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Action } from '../Actions';
+import ActionBar from '../ActionBar';
+import InputTitleSubHeader from './InputTitleSubHeader';
+import theme from './SubHeaderBar.scss';
+
+function SubHeaderBarActions({ components, left, right, center, className }) {
+	return (
+		<div className={className}>
```

Comment:
You could use `<nav />`

Reply:
The whole subheader has already a navbar. So it will render a nav in another nav.